tools/logcat-parse/Directory.Build.targets

```diff
@@ -0,0 +1,15 @@
+<Project>
+
+  <Target Name="_CopyDebugSymbols"
+      Condition=" '$(TargetFramework)' != '' "
+      AfterTargets="Build"
+      Inputs="$(PkgMono_Terminal)\lib\netstandard2.0\LineEditor.pdb"
+      Outputs="$(OutputPath)\LineEditor.pdb" >
```

Comment:
I think for these `Inputs` / `Outputs` to work, you would need a `<Touch/>`. The destination file will have the same last write time as the source file when using `<Copy/>`.

Since you are also using `SkipUnchangedFiles="true"`, another option is to just remove the `Inputs` / `Outputs` because you probably don't need both.